rename-files.test.js

```diff
@@ -0,0 +1,75 @@
+const fs = require('fs');
+const { createFiles } = require('./index');
+const { readDirectory, rename, getModifiedTime } = require('./rename-files');
```

Comment:
missing an import here! readEverything. Check your rename-files.js where you are exporting your functions.